frontend/views/containers/dashboard/GroupMembersAllModal.vue

```diff
@@ -113,6 +134,15 @@ export default {
     }
   },
   methods: {
+    isNewMember (username) {
+      if (username === this.ourUsername) {
+        return false
+      }
+      const weJoined = this.currentGroupState.profiles[this.ourUsername].joined_ms
+      const memberJoined = this.currentGroupState.profiles[username].joined_ms
+      const joinedAfterUs = weJoined < memberJoined
+      return joinedAfterUs && this.dateNow - memberJoined < 604800000 // joined less than 1w (168h) ago.
```

Comment:
Use `moment` on the ISOString

Reply:
I'm not familiar with `moment`. What method do you think it's the best for this case?

Reply:
- https://momentjs.com/docs/#/manipulating/
- https://momentjs.com/docs/#/parsing/date/
- https://momentjs.com/docs/#/parsing/string/

You can `import moment from 'moment'` at the top (it seems to come from one of our dependencies... not sure how)

Reply:
Hum... it seems an elephant for what we need here. But found a way more or less intutive:

```js
moment(memberJoined).isBefore(this.now, 'week')
```

this.now is a computed property returning `moment()`. It should be a computed property to avoid "calculating" now N times in a N loop. Specially on the list with all members (and search).

Btw, I looked into our source and `moment` is an indirect development dependency (it means we don't import it on package.json). The package importing it is Cypress!

So... if we really want to use `moment` it should be part of our dependencies. Just a reminder that this package is [65kb minified](https://bundlephobia.com/result?p=moment@2.24.0). I believe it can be useful to display a more "human time"  on proposals and chat... but it's way heavier than I thought.

Reply:
I agree that `moment` is probably something we can live without. If you would like to switch back to using the microseconds approach feel free, just initialize the date object using the string (`new Date(timestamp).getTime()`) and then you can use it with `Date.now()`.

I still do not think a computed property is needed. The caching isn't needed in this case I do not think.

Reply:
I'm okay with leaving it with `moment`. For now, it's more than okay. I just wanted to show my concern...

> I still do not think a computed property is needed. The caching isn't needed in this case I do not think.

Well, it's the difference between calculating "Date.now()" n times vs 1 time. When n times is the number of members and every time we do a search. I think some cache doesn't hurt.

Reply:
> Well, it's the difference between calculating "Date.now()" n times vs 1 time

That's not an issue since `Date.now()` doesn't take any significant resources.  Also, we want to make sure that it's updated eventually and isn't cached forever, which is what might happen with `this.now` since that doesn't depend on any Vue.js variables, and so it might return the same value forever.

Reply:
If you can implement this without `moment` it would safe us time in the future too, since we want to get removing unnecessary dependencies the closer we get to releasing this.

Reply:
> Also, we want to make sure that it's updated eventually and isn't cached forever

That's not how component's computed properties work. They are "cached" only while the component is mounted. Once it's destroyed (ex: the page changes or the modal is closed), it will re-calculate everything again.

But anyway... changed back to "old school" and without using a custom property. We have other performance issues more important than this micro-optimization ;)